guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: single
+title:  "Deploy Helm Charts"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.8" %}
+
+{% include toc %}
+
+Spinnaker surfaces a "Bake (Manifest)" stage to turn templates into manifests
+with the help of a templating engine. Currently, the only supported templating
+engine is [Helm](https://helm.sh/), by relying the `helm template` command. See
+more details [here](https://docs.helm.sh/helm/#helm_template).
+
+> Note: Make sure that you have configured [artifact support](/setup/artifacts)
+> in Spinnaker first. All Helm charts are fetched/stored as artifacts in
+> Spinnaker. Read more in the [reference pages](/reference/artifacts).
+
+## Configure the "Bake (Manifest)" stage
+
+When configuring the "Bake (Manifest)" stage, you can configure
+the following:
+
+* __The release name__ (required)
+
+  The Helm release name for this chart, and informs the name of the
+  artifact produced by this stage.
+
+* __The template artifact__ (required)
+
+  The Helm chart that you will be deploying, stored remotely as a
+  `.tar.gz` archive. You can produce this by running `helm package
+  /path/to/chart`. See more details
+  [here](https://docs.helm.sh/helm/#helm-package).
+
+* __Zero or more override artifacts__ (optional)
+
+  The files passed to `--values` parameter in the [`helm
+  template` command](https://docs.helm.sh/helm/#helm_template). Each is a
+  remotely stored artifact representing a [Helm Value
+  File](https://docs.helm.sh/chart_template_guide/#values-files).
+
+* __Statically specified overrides__
+
+  The set of static of key/value pairs that are passed as `--set` parameters to
+  the [`helm template` command](https://docs.helm.sh/helm/#helm_template).
+
+As an example, we have a fully configured Bake (Manifest) stage below:
+
+{%
+  include
+  figure
+  image_path="./bake-manifest-stage.png"
+%}
+
+Notice that in the "Produces Artifacts" section, Spinnaker has automatically
+created an `embedded/base64` artifact that will be bound when the stage
+completes, representing the fully baked manifest set to be deployed downstream.
```

Comment:
Is "manifest set" a term that readers are likely to understand, or would a short explanation be useful? I'm assuming this is actually one or more files - does the artifact actually represent a folder? I.e. do you get all the manifests if you reference that artifact downstream? Or can you get only some of them?